client/landing/gutenboarding/hooks/use-site-title.ts

```diff
@@ -13,11 +13,14 @@ import usePodcastTitle from './use-podcast-title';
 export default function useSiteTitle(): void {
 	const { setSiteTitle } = useDispatch( ONBOARD_STORE );
 	const podcastTitle = usePodcastTitle();
+	const { siteTitle } = useSelect( ( select ) => ( {
+		siteTitle: select( ONBOARD_STORE ).getSelectedSiteTitle(),
```

Comment:
Following the performance advice from p7H4VZ-2Q6-p2, what about changing this to a boolean, `hasSiteTitle`? As far as I can tell, we don't need to know the actual title here, just if it's set or not.

Reply:
Thanks, I missed that! I made the change in https://github.com/Automattic/wp-calypso/pull/50148/commits/52a87ebd8707f9f8f349a2b8047f2f0eb910e7ab